package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "hardhat --network localhost test",
+    "lint init": "find contracts/ -iname *.cairo -exec .venv/bin/cairo-format -i {} +",
```

Comment:
this line checks and fixes cairo files, right? a good name could be lint:fix. usually we add ":" in script names to add another behavior for the same step (compilation, tests, lint, deployment, etc.). see scripts that start with "compile:" few lines below

Reply:
fixed